src/config.ts

```diff
@@ -75,6 +75,8 @@ export interface SnowpackPlugin {
   output: string[];
   /** transform input to output */
   build?(options: BuildOptions): Promise<BuildResult | null>;
+  /** DEPRECATED */
+  transform?(options: BuildOptions): Promise<{result: string} | null>;
```

Comment:
🤔 what if we left this off, and simply overrode the type above? Just to prevent other contributors to making an external plugin with this API, etc.

Reply:
👍 makes sense